test/Driver/sanitizers.swift

```diff
@@ -20,6 +26,24 @@
 // RUN: not %swiftc_driver -resource-dir %S/Inputs/fake-resource-dir/lib/swift/ -driver-print-jobs -sanitize=thread -target x86_64-unknown-windows-msvc %s 2>&1 | %FileCheck -check-prefix=TSAN_WINDOWS %s
 // RUN: %swiftc_driver -resource-dir %S/Inputs/fake-resource-dir/lib/swift/ -driver-print-jobs -sanitize=thread -target x86_64-unknown-linux-gnu %s 2>&1 | %FileCheck -check-prefix=TSAN_LINUX %s
 
+/*
+ * Undefined Behavior Sanitizer Tests  (ubsan)
+ */
+
+// RUN: %swiftc_driver -resource-dir %S/Inputs/fake-resource-dir/lib/swift/ -driver-print-jobs -sanitize=undefined -target x86_64-apple-macosx10.9 %s | %FileCheck -check-prefix=UBSAN -check-prefix=UBSAN_OSX %s
+// RUN: %swiftc_driver -resource-dir %S/Inputs/fake-resource-dir/lib/swift/ -driver-print-jobs -sanitize=undefined -target x86-apple-macosx10.9 %s 2>&1 | %FileCheck -check-prefix=UBSAN_OSX_32 %s
```

Comment:
The positive checks should be checking both `-check-prefix=UBSAN` and `-check-prefix=UBSAN_<OS>`.

Reply:
Fixing! Is there a reason the Address Sanitizer Linux / Windows positive checks don't specify `-check-prefix=ASAN` ?

Reply:
I suspect it was just a copy/paste mistake, like yours were.

Reply:
Ah, wait, no, the `rpath` command is Apple-platform-specific. I guess it's okay to leave those as they were.

Reply:
Makes sense, I'll address and run the test suite locally before updating